app/boards/shields/pteron36/pteron36.conf

```diff
@@ -0,0 +1,10 @@
+
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment the following line to enable the Lotus58 OLED Display
```

Comment:
Might want to remove the display and encoder configs to not confuse users, if they aren't supported by the firmware yet.